samples/csharp_dotnetcore/53.teams-messaging-extensions-action-preview/TeamsMessagingExtensionsActionPreview.csproj

```diff
@@ -8,7 +8,7 @@
   <ItemGroup>
     <PackageReference Include="AdaptiveCards" Version="1.0.0" />
```

Comment:
Not part of your changes, but this version of AdaptiveCards differs from sample [54.teams-task-module](https://github.com/microsoft/BotBuilder-Samples/pull/2697/files#diff-8cfcd63486a3b3b65cb9bfd3926c0c21R9) (1.2.0).

Do you know if there's a reason they're not the same?